cmd/ixod/cmd/testnet.go

```diff
@@ -189,8 +190,10 @@ func InitTestnet(
 			return err
 		}
 
-		accTokens := sdk.TokensFromConsensusPower(1000)
-		accStakingTokens := sdk.TokensFromConsensusPower(500)
+		// TODO: Is default Power reduction correct?
+		accTokens := sdk.TokensFromConsensusPower(1000, sdk.DefaultPowerReduction)
+		// TODO: Is default Power reduction correct?
+		accStakingTokens := sdk.TokensFromConsensusPower(500, sdk.DefaultPowerReduction)
```

Comment:
@ig-shaun @demondayza @steveixo 
`TokensFromConsensusPower` This function now requires the power reduction argument, I have used the sdk default which is `var DefaultPowerReduction = NewIntFromUint64(1000000)`

Reply:
Here we will need @ig-shaun to chip in on the default as its also a business decision.